fluentd/lib/fluent-plugin-prometheus/in_prometheus_tail_monitor.rb

```diff
@@ -0,0 +1,120 @@
+require 'fluent/plugin/input'
```

Comment:
This file should be either removed entirely, appear as a patch to vendored_src, are a new file to vendored_src since you cant patch it

Reply:
@jcantrill  i would like to know where we can keep this file version controlled as a single piece of truth to all changes that are used to create patch over vendored_src baseline implementation of this plugin.